### DISCUSSION TITLE
Errors in file cellseg_models.pytorch/examples/pannuke_nuclei_segmentation_stardist.ipynb

### DISCUSSION THREAD

**Discussion by sunyoung0509** [okunator/cellseg_models.pytorch#55] [General]
I get an error saying "AttributeError: type object 'FileHandler' has no attribute 'write_mask'" 
in the file pannuke_datamodule.py I also get to know that there is no deffinition of write_mask. Was there any updates? Or didn't you have the same problem?

**Comment by arjunreddy25** [comment]
have you got this error 
AttributeError: 'WindowsPath' object has no attribute 'decode'
